### PR TITLE
lesson-3.1

### PR DESCRIPTION
**Pull request by zemiarsi**

### CODE REVIEW

File: lesson_1.1/src/AddTodoForm.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import React, { useState } from "react";
 import InputWithLabel from "./InputWithLabel";
+import Button from '@mui/material/Button'
+import AddBoxIcon from '@mui/icons-material/AddBox';
+// import AddBoxOutlinedIcon from '@mui/icons-material/AddBoxOutlined';
```
Comment (kaytijackson97) [inline lesson_1.1/src/AddTodoForm.js:5]: You can remove unused commented out lines of code

File: lesson_1.1/src/AddTodoForm.js [hunk 2]
```diff
@@ -15,15 +22,18 @@ function AddTodoForm({ onAddTodo }) {
...
+        <InputWithLabel 
           todoTitle={todoTitle}
           handleTitleChange={handleTitleChange}
         >
           Title:
         </InputWithLabel>
-        <button type="submit">Add</button>
+        &nbsp;
+        <Button className={style.addBtn} onClick={() => console.log('you clicked me')} type="submit">
+        <AddBoxIcon  fontSize='medium'/>
```
Comment (kaytijackson97) [inline lesson_1.1/src/AddTodoForm.js:35]: should be indented one place

File: lesson_1.1/src/AddTodoForm.js [hunk 3]
```diff
@@ -15,15 +22,18 @@ function AddTodoForm({ onAddTodo }) {
...
       <form onSubmit={handleAddTodo}>
-        <InputWithLabel
+        <InputWithLabel 
           todoTitle={todoTitle}
           handleTitleChange={handleTitleChange}
         >
           Title:
         </InputWithLabel>
-        <button type="submit">Add</button>
+        &nbsp;
```
Comment (kaytijackson97) [inline lesson_1.1/src/AddTodoForm.js:33]: not sure what this line's purpose is, but don't think it's necessary

File: lesson_1.1/src/App.js [hunk 4]
```diff
@@ -1,82 +1,63 @@
...
-            <p>Loading...</p>
-          ) : (
-            <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
-          )}
+        
+        <Route  path="/list-1">
+          <TodoContainer tableName="CTD"/>
+        </Route>
+        <Route path="/list-2">
+        <TodoContainer tableName="Mentor"/>
```
Comment (kaytijackson97) [inline lesson_1.1/src/App.js:44]: should be indented

File: lesson_1.1/src/App.js [hunk 5]
```diff
@@ -1,82 +1,63 @@
...
+        </Route>
+        <Route path="/list-3">
+        < TodoContainer tableName="Personal"/>
```
Comment (kaytijackson97) [inline lesson_1.1/src/App.js:47]: should be indented

File: lesson_1.1/src/App.js [hunk 6]
```diff
@@ -1,82 +1,63 @@
...
         </Route>
-        <Route path="/new">
-          <h1>New Todo List</h1>
+        <Route path="/">
+          <p>select a list...</p>
+          <div className="CTDimg">
+      <a target="_blank" href="CTD.png">
```
Comment (kaytijackson97) [inline lesson_1.1/src/App.js:52]: line 52 to 56 indentation should be fixed.

File: lesson_1.1/src/TodoContainer.js [hunk 7]
```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect }from 'react';
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
+// import { style } from '@mui/system';
```
Comment (kaytijackson97) [inline lesson_1.1/src/TodoContainer.js:4]: can remove unused commented lines of code

File: lesson_1.1/src/TodoListItem.js [hunk 8]
```diff
@@ -1,13 +1,23 @@
...
   console.log({ todo });
   return (
     <div>
-      <li >
-        {todo.fields.Title}
-        <button onClick={() => onRemoveTodo(todo.id)} type="button">
-          Remove
+      <li className={style.ListItem} >
+        <input type='checkbox' fontSize='medium' className={style.CheckBox}/> 
+       {/* <Checkbox className={style.Checkbox}/>  */}
```
Comment (kaytijackson97) [inline lesson_1.1/src/TodoListItem.js:13]: can remove unused commented out lines of code

File: lesson_1.1/src/TodoListItem.js [hunk 9]
```diff
@@ -1,13 +1,23 @@
...
+      <span className={ todo.fields.completed===true? style.CompletedListItem:''}>
```
Comment (kaytijackson97) [inline lesson_1.1/src/TodoListItem.js:14]: lines 14 to 16 indentation should be fixed

File: lesson_1.1/src/TodoListItem.module.css [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+  text-align: center; 
+  font-size: 20px;
+} 
+
+.CompletedListItem {
+  text-decoration: line-through;
+}
+
+.CheckBox {
+ color:white;
```
Comment (kaytijackson97) [inline lesson_1.1/src/TodoListItem.module.css:30]: inside code should be indented

**Review by kaytijackson97 — COMMENTED** [review]
Thank you so much for being patient as I've been away for recent holidays. Functionality is really good! Just make sure to keep an eye on removing unused code to keep clean and also maintaining good indentation to visually keep up organization. Couple fixes and this code will be in top shape.

File: lesson_1.1/src/App.js [hunk 11]
```diff
@@ -1,82 +1,63 @@
...
+          <li className={style.navLi}>
+            <Link to="/list-2" className={style.navHref}><VolunteerActivismRoundedIcon  color="primary"/>Mentor</Link>
+          </li>
+          <li className={style.navLi}>
+            <Link to="/list-3" className={style.navHref}><PersonRoundedIcon  color="primary"/>Personal</Link>
+          </li>
+        </ul>
+        
+      </nav>
+
```
Comment (kaytijackson97) [inline lesson_1.1/src/App.js:36]: A lot of white space in this return statement that isn't needed

**Comment by zemiarsi** [comment]
No problem, I hope u enjoyed your holiday!! Okay I will fixing that one, Thanks !!

**Pull request merged**